src/material/tooltip/testing/tooltip-harness.ts

```diff
@@ -31,8 +24,7 @@ export class MatTooltipHarness extends ComponentHarness {
     // We need to dispatch both `touchstart` and a hover event, because the tooltip binds
     // different events depending on the device. The `changedTouches` is there in case the
     // element has ripples.
-    // @breaking-change 12.0.0 Remove null assertion from `dispatchEvent`.
-    await host.dispatchEvent?.('touchstart', {changedTouches: []});
+    await host.dispatchEvent('touchstart', {changedTouches: []});
```

Comment:
does it need to be `target: major` because of this?

Reply:
That's a good point. I'll put the null checks back in.